CMakeLists.txt

```diff
@@ -2,19 +2,19 @@ CMAKE_MINIMUM_REQUIRED(VERSION 3.0.2)
 
 SET(FC "/usr/bin/gfortran" CACHE STRING "Fortran Compiler")
 SET(FFLAGS "-cpp" CACHE STRING "Fortran compiler flags")
-SET(FTOBJ_PREFIX "/usr/local/ftobjectlibrary" CACHE STRING "Path to the FTOBJ installation path")
 
 MESSAGE("---------------  cmake START -------------------")
 MESSAGE("-- FC:                   ${FC}")
 MESSAGE("-- FFLAGS:               ${FFLAGS}")
-MESSAGE("-- FTOBJ_PREFIX:         ${FTOBJ_PREFIX}")
 MESSAGE("-- CMAKE_INSTALL_PREFIX: ${CMAKE_INSTALL_PREFIX}")
 MESSAGE("------------------------------------------------")
 
 SET(CMAKE_Fortran_COMPILER_INIT ${FC})
 SET(CMAKE_Fortran_FLAGS "${FFLAGS} ${CMAKE_Fortran_FLAGS}")
 
-INCLUDE_DIRECTORIES("${FTOBJ_PREFIX}/include")
+FIND_PACKAGE(FTOBJECTLIBRARY REQUIRED)
+
+INCLUDE_DIRECTORIES("${FTOBJECTLIBRARY_INCLUDE_DIRS}")
 
 PROJECT(HOHQMesh Fortran)
 SET(VERSION 0.0.0)
```

Comment:
In the official release tarballs is a file `VERSION` that holds the version (currently `1.0.0`). You could check for the existence of this file and use that version instead of `0.0.0`.